deployment/media-insights-stack.yaml

```diff
@@ -468,6 +467,44 @@ Resources:
                 Resource: "*"
 
 
+  WorkflowExecutionStreamLambdaRole:
+    Type: "AWS::IAM::Role"
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service:
+                - lambda.amazonaws.com
+            Action:
+              - sts:AssumeRole
+      Policies:
+        - PolicyName: !Sub "${AWS::StackName}-WorkflowExecutionLambdaStreamAccessPolicy"
+          PolicyDocument:
+            Statement:
+              - Effect: Allow
+                Action:
+                  - "dynamodb:DescribeStream"
+                  - "dynamodb:GetRecords"
+                  - "dynamodb:GetShardIterator"
+                  - "dynamodb:ListStreams"
+                Resource: 
+                  Fn::GetAtt:
+                    - WorkflowExecutionTable
+                    - StreamArn
+              - Effect: Allow
+                Action:
+                  - "sns:Publish"
+                Resource: 
+                  !Ref WorkflowExecutionEventTopic
+              # TODO: Security - Scope this down
+              - Effect: Allow
+                Action:
+                  - "logs:CreateLogGroup"
+                  - "logs:CreateLogStream"
+                  - "logs:PutLogEvents"
+                Resource: "*"
```

Comment:
Might need to add a cfn_nag exception for this.

Reply:
I found this cfn_nag from content analysis code which should do the trick:
```
rules_to_suppress:
          - id: W11
            reason: "Lambda requires ability to write to cloudwatch *, as configured in the default AWS lambda execution role."
```

Reply:
done